temporian/implementation/numpy/data/display_utils.py

```diff
@@ -77,6 +77,10 @@ def display_html(evset: EventSet) -> str:
         index_data = evset.data[index_key]
         num_timestamps = len(index_data.timestamps)
         max_timestamps = config.display_max_events or num_timestamps
+        #Slices timestamps and features if there are more than 7 events
+        half_max_timestamps = (max_timestamps // 2)
+        display_timestamps = np.concatenate((index_data.timestamps[:half_max_timestamps], index_data.timestamps[-half_max_timestamps:])) if num_timestamps > max_timestamps else index_data.timestamps[:num_timestamps]
```

Comment:
I think the `[:num_timestamps]` at the end of this line might be unnecessary? won't it always be returning all the timestamps?